packages/core/src/app/address/isValidCustomerAddress.ts

```diff
@@ -1,12 +1,12 @@
-import { Address, CustomerAddress, FormField } from '@bigcommerce/checkout-sdk';
+import { Address, FormField } from '@bigcommerce/checkout-sdk';
 import { some } from 'lodash';
 
 import isEqualAddress from './isEqualAddress';
 import isValidAddress from './isValidAddress';
 
 export default function isValidCustomerAddress(
     address: Address | undefined,
-    addresses: CustomerAddress[],
+    addresses: Address[],
```

Comment:
This feels odd as we are changing the address interface that is passed in, and hence essentially changing the method.